inference-engine/tests/functional/plugin/shared/src/single_layer_tests/mvn.cpp

```diff
@@ -54,6 +54,13 @@ void MvnLayerTest::SetUp() {
 
 TEST_P(MvnLayerTest, CompareWithRefs) {
     Run();
-};
+}
+
+TEST_P(MvnLayerTest, TestForDynamicBatch) {
+    if (targetDevice == CommonTestUtils::DEVICE_GPU || targetDevice == CommonTestUtils::DEVICE_CPU) {
```

Comment:
IMO, we should create instance only with config as parameter for CPU and GPU plugins. Creation of this test case looks like extra action.